src/rpc/rpc_reward_api.py

```diff
@@ -58,12 +60,12 @@ def get_rewards_for_cycle_map(self, cycle, expected_rewards=False):
                          .format(cycle, self.preserved_cycles, self.blocks_per_cycle,
                                  level_of_first_block_in_preserved_cycles, level_of_last_block_in_unfreeze_cycle))
 
-            # Decide on if paying actual rewards earned, or paying expected/ideal rewards
-            if expected_rewards:
+            # Decide on if paying actual rewards earned, or paying estimated/ideal rewards
+            if rewards_type.isEstimated():
 
                 # Determine how many priority 0 baking rights delegate had
-                nb_blocks = self.__get_number_of_baking_rights(cycle, level_of_first_block_in_preserved_cycles)
-                nb_endorsements = self.__get_number_of_endorsement_rights(cycle, level_of_first_block_in_preserved_cycles)
+                nb_blocks = len([r for r in self.__get_baking_rights(cycle, level_of_first_block_in_preserved_cycles) if r["priority"] == 0])
```

Comment:
Why did you change this?

Reply:
The function `get_number_of_baking_rights` doesn't exist anymore, and has been replaced by `get_baking_rights`.

On master branch, I only need to count baking rights. But on this PR, I need to match the list of baking rights with the actual bakes, so I can tell which bakes were successful and which ones were not.

On this line though, I just need to count the rights. Before, the count was done inside the function, now it is done outside of the function.